capsul/pipeline/pipeline.py

```diff
@@ -938,8 +924,8 @@ def update_nodes_and_plugs_activation(self):
                 if test:
                     for plug_name, plug in test:
                         if debug:
-                            print >> debug, '%d-%s:%s' % (
-                                iteration, node.full_name, plug_name)
+                            print >> debug, '%d-%s:%s' % (iteration, node.full_name, plug_name)
+                        #print '!deactivations! iteration', iteration, '--- %s:%s' % (node.full_name,plug_name)
```

Comment:
Why this commented print is inserted here. It looks like the kind of debug lines that I use sometimes. Is it a line that have been removed in a previous changeset and that is reintroduced here ?